app/dependencies/AuthzStore.go

```diff
@@ -0,0 +1,9 @@
+package dependencies
```

Comment:
I am not sure about the module dependencies. Please can you elaborate the purpose?